app/boards/arm/adv360pro/adv360pro-pinctrl.dtsi

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2023 The ZMK Contributors
+ * SPDX-License-Identifier: MIT
+ */
+
+&pinctrl {
+    spi3_default: spi3_default {
+        group1 {
+            psels = <NRF_PSEL(SPIM_MOSI, 0, 20)>;
+        };
+    };
+
+    spi3_sleep: spi3_sleep {
+        group1 {
+            psels = <NRF_PSEL(SPIM_MOSI, 0, 20)>;
+            low-power-enable;
+        };
+    };
+    pwm0_default: pwm0_default {
+        group1 {
+            psels = <NRF_PSEL(PWM_OUT0, 0, 17)>;
+        };
+    };
+    pwm0_sleep: pwm0_sleep {
+        group1 {
+            psels = <NRF_PSEL(PWM_OUT0, 0, 17)>;
+            low-power-enable;
+        };
+    };
+};
```

Comment:
Nit: does this project care about consistent file line endings?

Reply:
I don't think we have a pre-commit check on that, and in practice it is hard to enforce without it (and it hasn't been, so far).